backend/scripts/monitor_ai_processing.py

```
@@ -0,0 +1,338 @@
+            else:
+                return "empty_content"
+        else:
+            return "failed"
+
+    def _analyze_metadata_status(self, ai_result: AIResult = None) -> str:
+        """分析元数据处理状态"""
+        if not ai_result:
+            return "no_ai_result"
+        
+        # 检查是否有基本的元数据
+        if (ai_result.labels or 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)